gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineIdea.groovy

```diff
@@ -18,9 +18,17 @@ package com.palantir.baseline.plugins
 
 import com.google.common.collect.ImmutableMap
 import com.palantir.baseline.IntellijSupport
+import com.palantir.baseline.extensions.BaselineJavaVersionExtension
+import com.palantir.baseline.extensions.BaselineJavaVersionsExtension
 import com.palantir.baseline.util.GitUtils
 import groovy.transform.CompileStatic
 import groovy.xml.XmlUtil
+import org.gradle.api.JavaVersion
+import org.gradle.jvm.toolchain.JavaLanguageVersion
+import org.gradle.plugins.ide.idea.model.IdeaLanguageLevel
+import org.gradle.plugins.ide.idea.model.IdeaModule
+import org.gradle.plugins.ide.idea.model.Jdk
+
```

Comment:
what happened with the imports here?

Reply:
We don't have a formatter for groovy that automatically cleans up imports.